### PR TITLE
Feature 163666888 support req and res

### PR DESCRIPTION
**Pull request by sandinmyjoints**
@ahaurw01 

I believe these are the last pieces of winex that Neodarwin uses that pn-logging did not support.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -97,23 +97,23 @@ describe('index', function() {
...
 
     it('should include default meta in log messages', function() {
-        log = new index.Log({
-          transports: [
-            {
-              TestTransport: {
-                level: 'debug',
-                __log: spy,
-              },
+      log = new index.Log({
```
Comment (sandinmyjoints) [inline test/index.js:100]: This diff hunk is prettier.

**Review by ahaurw01 — CHANGES_REQUESTED** [review]
I don't think the addition of `req` and `res` like this quite fits the different API that pn-logging exposes. 

Projects that use pn-logging typically have one `Log` instance that is created using the configured options for the environment. For example, sd-playground has a `logging` module that exports a `new Log(config.get('logging'))`. Then that singleton instance is used throughout the project to fire off `.info(...)`, `.error(...)`, etc. We'd be at risk of having one `req` object get applied to totally unrelated log entries, I believe. If neodarwin doesn't use a singleton then we might be ok in practice, but it might lead to confusion and bugs in the future.

Maybe instead we extend the logging methods somehow? E.g. `log.info(msg, meta, error, req, res)`.

**Comment by sandinmyjoints** [comment]
Thanks -- you're right, this is a winex solution but not right for pn-logging. I'll rework it.

**Comment by sandinmyjoints** [comment]
@ahaurw01 BTY.

**Review by ahaurw01 — APPROVED** [review]
Looks good!

**Pull request merged**